packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -464,12 +464,20 @@ def get(self, track_id):
         abort_not_found(track_id, ns)
 
 
+# Comments
+track_comments_args = pagination_parser.copy()
```

Comment:
nice, lets use the pagination_with_current_user parser though to maintain api patterns?

Reply:
oh nice, I somehow missed this, muche asier